config/routes.rb

```diff
@@ -2,7 +2,7 @@
   devise_for :users
   resources :users do
     resources :bookings, except: [:index, :show, :destroy, :edit, :update]
-    resources :reviews, except: [:index, :show, :destroy]
+    resources :reviews, except: [:index, :show, :destroy, :edit, :update]
```

Comment:
I think `only: [:create]` should do it. Don't think we need `:new` as we are planning on displaying the 'new comment' form at the bottom of the artists page?